charts/nginx-gateway-fabric/values.yaml

```
@@ -134,6 +134,29 @@ nginx:
   # -- Is NGINX Plus image being used
   plus: false
 
+  # Configuration for NGINX Plus usage reporting.
+  usage:
+    # -- The name of the Secret containing the JWT for NGINX Plus usage reporting. Must exist in the same namespace
+    # that the NGINX Gateway Fabric control plane is running in (default namespace: nginx-gateway).
+    secretName: "nplus-license"
+
+    # -- The endpoint of the NGINX Plus usage reporting server. Default: product.connect.nginx.com
+    endpoint: ""
+
+    # -- The resolver domain name or IP address with optional port for resolving the endpoint.
+    resolver: ""
+
+    # -- Disable client verification of the NGINX Plus usage reporting server certificate.
+    skipVerify: false
+
+    # -- The name of the Secret containing the CA cert for verifying the NGINX Plus usage reporting
```

nit:

Do you want to use the full word?

```suggestion
    # -- The name of the Secret containing the CA certificate for verifying the NGINX Plus usage reporting
```